sources/TileDB.CSharp/Group.cs

```diff
@@ -252,18 +258,18 @@ public ulong MemberCount()
         /// </summary>
         /// <param name="index"></param>
         /// <returns></returns>
-        (string uri, ObjectType object_type, string name) MemberByIndex(ulong index) 
+        (string uri, ObjectType object_type, string name) MemberByIndex(ulong index)
```

Comment:
Good catch on the typo! I think we could also make this method public and then add to `tests/TileDB.CSharp.Test/GroupTest.cs` -

```C#
//Reopen in read mode
group1.Open(QueryType.TILEDB_READ);
Assert.AreEqual<ulong>(2, group1.MemberCount());
Assert.AreEqual("array2", group1.MemberByIndex(0).name);
Assert.AreEqual("array1", group1.MemberByIndex(1).name);
group1.Close();
```

I did this locally using this branch and the test passes, just thinking while we're here it would be nice to test for this to protect against future regressions.

Reply:
I prefer to handle it separately. The API signature also needs some improvements.